src/core/Akka.Tests/Actor/ActorRefSpec.cs

```diff
@@ -73,6 +74,7 @@ public void An_ActorRef_should_not_equal_another_ActorRef_when_path_differs()
         public void An_ActorRef_should_not_allow_actors_to_be_created_outside_an_ActorOf()
         {
             Shutdown();
+            InternalCurrentActorCellKeeper.Current = null;
```

Comment:
Hmm... if the test fails without this row, we need to fix this somewhere else. Did it start failing by switching to xUnit 2?